### PR TITLE
feat: updated the backend book model to capture the author 'property' instead of the 'userId' property. Testing 'console.logs' from the controller directory were removed.

### PR DESCRIPTION
**Pull request by alexh205**
image

### CODE REVIEW

File: backend/.env.example [hunk 1]
```diff
@@ -3,3 +3,4 @@ DB_URI='mongodb://localhost:27017/your-database-name'
 BREVO_USERID='your-brevo-user-id'
 BREVO_PASS='your-brevo-password'
 JWT_SECRET='your-jwt'
+DEMOUSER_PASSWORD='password'
```
Comment (jackie-leary) [inline backend/.env.example:6]: What is demouser?

Comment (alexh205) [reply backend/.env.example:6]: This is will be a env key to hold the Demo Users's password value. The demo user will provide access to the backend without needing to create a new user during development. The demouser should be part of the PR that Jeison is working on 'database seeding'

File: backend/models/Books.js [hunk 2]
```diff
@@ -1,70 +1,64 @@
...
+			'Health',
+		],
+		trim: true,
+	},
+	author: {
+		type: String,
+		required: [true, 'author is required'],
+		minlength: 6,
+	},
+	isBan: {
```
Comment (jackie-leary) [inline backend/models/Books.js]: I think `isBan` can be removed. @alexh205

Comment (alexh205) [reply backend/models/Books.js]: The 'isBan' property has been updated to 'ISBN'.

File: backend/models/Books.js [hunk 3]
```diff
@@ -1,70 +1,64 @@
...
+			'Mystery',
+			'Self-Help',
+			'Health',
+		],
+		trim: true,
+	},
+	author: {
+		type: String,
+		required: [true, 'author is required'],
+		minlength: 6,
```
Comment (jackie-leary) [inline backend/models/Books.js:37]: nitpick: an author could have a really short name

**Review by jackie-leary — APPROVED** [review]
left another comment, but it's small so approving!

**Pull request merged**